### PR TITLE
Refs #31055 -- Doc'd 'databases' argument of check functions.

### PR DESCRIPTION
**Pull request by adamchainz**
Added in #12396 but not covered for the custom checks.

### CODE REVIEW

**Comment by adamchainz** [comment]
Rebased

File: docs/topics/checks.txt [hunk 1]
```diff
@@ -48,8 +48,14 @@ check function::
...
 The check function *must* accept an ``app_configs`` argument; this argument is
 the list of applications that should be inspected. If ``None``, the check must
-be run on *all* installed apps in the project. The ``**kwargs`` argument is
-required for future expansion.
+be run on *all* installed apps in the project.
+
+The check will also receive a ``databases`` argument, which is a list of
+database aliases whose connections may be used to inspect database level
+configuration. If this is ``None``, the check must not use any database
+connections.
```
Comment (carltongibson) [inline docs/topics/checks.txt:56]:
```suggestion
The check will receive a ``databases`` keyword argument. This is a list of
database aliases whose connections may be used to inspect database level
configuration. If ``databases`` is ``None``, the check must not use any
database connections.
```

**Review by carltongibson — COMMENTED** [review]
Hey @adamchainz. 

This looks good thank you. I've suggest a tweak. Let me know if you're happy with that. 
(Don't mind if you rebase 😉)

**Comment by felixxm** [comment]
@adamchainz Thanks :+1:

**Pull request merged**